core/src/types/eth_abi.rs

```diff
@@ -102,13 +101,9 @@ pub trait Encode<const N: usize>: Sized {
     /// Encodes a slice of [`Token`] instances, and returns the
     /// keccak hash of the encoded string appended to an Ethereum
     /// signature header. This can then be signed.
-    fn signable_keccak256(&self) -> Vec<u8> {
-        let mut output = [0; 32];
-        let message = self.encode().into_inner();
-        let mut state = Keccak::v256();
-        state.update(&message);
-        state.finalize(&mut output);
-        SignableEthBytes::as_signable(&output)
+    fn signable_keccak256(&self) -> KeccakHash {
```

Comment:
way nicer now we use `KeccakHash` (`[u8; 32]`) now instead of `Vec<u8>`s throughout